src/main/resources/xslt/mods2solr.xsl

```diff
@@ -24,6 +24,10 @@
        <xsl:param name="access_ophavsperson_doedsaar"/>
        <xsl:param name="access_searlige_visningsvilkaar"/>
        <xsl:param name="access_materiale_type"/>
+       <xsl:param name="access_foto_aftale "/>
```

Comment:
The space at the end of the String needs to be removed.

Reply:
Correct.But somehow it seemed to work anyway... (XSL magic?)

Reply:
Probably due to a lenient XSLT implementation yes. And it would _probably_ also work with another implementation.